### DISCUSSION TITLE
Downgrade to GitHub Free - Would I still be able to have private repositories?

### DISCUSSION THREAD

**Discussion by enyangxxx** [community/community#23309] [New to GitHub]
<p>Currently I have the student pack, but it will expire soon… Once I am downgraded to the free plan, what would happen currently with my private repos, if I am not part of any team/organization? There is so much going on with Github pricing, I have no idea what is the current state.</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/enyangxxx">@enyangxxx</a>,</p>
<p>Not to worry! You will certainly still have private repositories and <a href="https://help.github.com/en/github/getting-started-with-github/githubs-products#github-free-for-user-accounts" title="GitHub Products: Free" rel="nofollow noopener">these other</a> features, as well after downgrading to Free. You may also want to review our new pinned topic where we’ve <a href="https://github.community/t/are-there-any-changes-to-github-pro/18546" title="Are there any changes to GitHub Pro">listed</a> the updated benefits of Pro, to be sure you’re not currently utilizing any features that Free does not offer.</p>

**Reply by davenpos** [reply]
That's good to hear. I recently got an email saying my GitHub pro coupon was about to expire, and it said in the email "Want to downgrade? You'll still have access to your private repositories on GitHub Free". I was afraid that meant I'd lose access to my public repositories after I get downgraded or that I wouldn't be able to make any more public repositories.

I don't actually remember getting this coupon. It must have been a couple years ago back when I was still a university student. Something else in the email that concerned me was this part:

"We noticed you haven’t entered any payment information yet. If you’d like to continue using (GitHub Pro - $4/month) you’ll need to add a valid payment method to your account. After your coupon expires, we’ll charge your account, and you can keep working without interruption."

Am I going to get charged the day the coupon expires? Do I absolutely have to enter in any payment information? Please clarify, I'm a little concerned.

**Comment by enyangxxx** [comment]
<p>Hi thanks! Does it mean, if i have a private repository with wiki pages, that repository will become invisible, as wiki is only a feature of the pro plan?</p>

**Comment by wordsoup** [comment]
<p>Hi <a class="mention" href="/enyangxxx">@enyangxxx</a>,</p>
<p>Your wiki would be disabled, and be available in the future if you choose to upgrade. I hope this helps.</p>
